css/style.css

```diff
@@ -0,0 +1,825 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.top-area {
+  display: flex;
+  background-color: #3e3e3e;
+  height: 40px;
+  justify-content: center;
+}
+
+.top-content {
+  display: flex;
+  width: 70%;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.social-icons-cart ul {
+  display: flex;
+}
+
+.social-icons-cart li {
+  list-style-type: none;
+}
+
+.social-icons-cart a {
+  text-decoration: none;
+  color: #fafafa;
+  margin-left: 25px;
+  font-family: 'Lato', sans-serif;
+  font-size: 14px;
+}
+
+.social-icons-cart .fa {
+  font-size: 18px;
+}
+
+.title-and-navigation {
+  display: flex;
+  justify-content: center;
+}
+
+.navigation-content {
+  width: 70%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.title {
+  display: flex;
+  width: 15%;
+}
+
+.nav {
+  width: 70%;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.navbar ul {
+  display: flex;
+}
+
+.navbar li {
+  list-style-type: none;
+}
+
+.navbar a {
+  text-decoration: none;
+  color: #999;
+  font-family: 'Lato', sans-serif;
+  padding: 10px 20px;
+  font-size: 15px;
+}
+
+.all-program-button a {
+  color: #fff;
+  text-decoration: underline;
+  font-family: 'Lato', sans-serif;
+  font-size: 16px;
+  text-transform: uppercase;
+}
+
+.contact a {
+  color: #353535;
+  margin-top: 10px;
+}
+
+.navbar a:hover {
+  color: #0e0e0e;
+  transition: all 1s;
+}
+
+.navbar li:last-child a {
+  border: 5px solid #ec3c24;
+  margin-left: 15px;
+}
+
+.mobile-menu {
+  display: none;
+}
+
+.hero-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+  background-color: #f7f7f7;
+  height: 100vh;
+  padding-bottom: 15%;
+}
```

Comment:
- [ ] At the desktop version there is extra padding at the bottom, kindly fix it to match it with the design guidelines for Behance
- [ ] Also in the mobile version the text for header section overlaping with the text of speaker section, kindly fix this issue also 
> your design
> ![image](https://user-images.githubusercontent.com/53402600/219468901-0d6eab2d-95cc-4c7b-8bd3-bb4b3a406a1f.png)